epifor/data/batch.py

```diff
@@ -289,15 +291,52 @@ def write_export_data(self, regions: Regions):
         country into the batch directory.
         """
 
-        out_dir = self.generate_export_dir()
+        out_dir = self.get_out_dir()
         out_json = out_dir / self.DATA_FILE_NAME
         ed = ExportDoc(comment=f"{self.name}")
 
-        for rkey in tqdm.tqdm(self.config["regions"], desc="Exporting regions"):
+        df = pd.read_hdf(f"{out_dir}/{self.HIST_FILE_NAME}")
+        columns_list = sorted(set([date.split('_')[1] for date in df.columns]))
```

Comment:
Since you are using hdf storage, you can just store 3 dataframes under different names in the same file, and use dates as columns directly.